client/lib/stats/stats-list/test/test-stats-list.js

```diff
@@ -1,124 +1,159 @@
-			assert.throw( function() { new StatList( { statType: 'statsWookieList', siteID: 1 } ); }, TypeError );
+		it( 'should require a statType', () => {
+			assert.throw( () => {
+				statListFactory( { siteID: 1 } );
+			}, TypeError );
 		} );
 
+		it( 'should only allow valid statTypes', () => {
+			assert.throw( () => {
+				statListFactory( { statType: 'statsWookieList', siteID: 1 } );
+			}, TypeError );
+		} );
 	} );
 
 	describe( 'attributes', function() {
+		let statList, errorStatList;
+		beforeEach( () => {
+			let options = { siteID: 1, statType: 'statsClicks', date: '2014-07-04', period: 'day' };
+			statList = new StatList( options );
 
-		it( 'should create a response object', function(){
+			let errorOptions = { siteID: 1, statType: 'statsEvents' };
+			errorStatList = new StatList( errorOptions );
+		} );
+
+		it( 'is always true', () => assert.isTrue( true ) )
+
+		it( 'should create a response object', () => {
 			assert.typeOf( statList.response, 'object', 'respose object created' );
 			assert.typeOf( statList.response.data, 'array', 'respose data array created' );
-
 		} );
 
-		it( 'should create an options object', function(){
+		it( 'should create an options object', () => {
 			assert.typeOf( statList.options, 'object', 'options object created' );
 		} );
 
-		it( 'should not have siteID in options object', function() {
+		it( 'should not have siteID in options object', () => {
 			assert.isUndefined( statList.options.siteID );
 		} );
 
-		it( 'should not have statType in options object', function() {
+		it( 'should not have statType in options object', () => {
 			assert.isUndefined( statList.options.statType );
 		} );
 
-		it( 'should set siteID', function(){
+		it( 'should set siteID', () => {
 			assert.equal( statList.siteID, 1, 'should set the siteID' );
 		} );
 
-		it( 'should set statType', function(){
+		it( 'should set statType', () => {
 			assert.equal( statList.statType, 'statsClicks', 'should set the statType' );
 		} );
 
-		it( 'should set localStoreKey', function(){
+		it( 'should set localStoreKey', () => {
 			assert.equal( statList.localStoreKey, statList.statType + statList.siteID, 'should set the localStoreKey' );
 		} );
 
-		it( 'should default performRetry to true', function(){
+		it( 'should default performRetry to true', () => {
 			assert.isTrue( statList.performRetry, 'should performRetry' );
 		} );
 
-		it( 'should set isDocumentedEndpoint correct', function() {
+		it( 'should set isDocumentedEndpoint correct', () => {
 			assert.isTrue( statList.isDocumentedEndpoint, 'documented endpoints should be true' );
 			assert.isFalse( errorStatList.isDocumentedEndpoint, 'undocumented endpoints should be false' );
 		} );
-
 	} );
 
 	describe( 'fetch', function() {
+		let statList, errorStatList;
 
-		it( 'should have a fetch function', function() {
+		before( () => {
+			let options = { siteID: 1, statType: 'statsClicks', date: '2014-07-04', period: 'day' };
```

Comment:
It looks like the stats-list mutates the options internally so if I move the options out to a parent `describe` I get failed tests.  So keeping each options instance scoped to the inner describe is probably best.

What I'll do is clone a common options object in each describe block